cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/survey/3-5/before-page-1.html

```diff
@@ -0,0 +1,3 @@
+<h2>How often do you do the following things</h2>
```

Comment:
Why are all these templates duplicated? The inclusion method would make sense if they were different, but since they're all the same, why not just bake this into `form-page.html`?

Reply:
We expect them to differ once the content is updated, and, the include paths are generated programmatically so easier to have a little duplication than complex logic to reuse templates.